dialogue-serde/src/main/java/com/palantir/conjure/java/dialogue/serde/ErrorDecoder.java

```diff
@@ -57,6 +59,9 @@ public enum ErrorDecoder {
     private static final SafeLogger log = SafeLoggerFactory.get(ErrorDecoder.class);
     private static final ObjectMapper MAPPER = ObjectMappers.newClientObjectMapper();
 
+    @VisibleForTesting
+    static final QosReason QOS_REASON = QosReason.of("client-qos-response");
```

Comment:
Wasn't sure what to use for the reason - I'm not particularly in favor of this name and open to suggestions.

Reply:
I don't have anything better in mind, we can likely change this later without breaking folks.